### DISCUSSION TITLE
does audiogridder support multichannel streams? specifficaly a 7.1.2 Dolby Atmos bed

### DISCUSSION THREAD

**Discussion by andreimiricescu** [apohl79/audiogridder#1314] [Help]
I'm trying to make this work with studio one in Dolby Atmos. every time I try inserting audio grinder on a 7.1.2. bed, it instantly crashes the daw. is there something I m doing wrong?

**Comment by th3funnychurch3s** [comment]
It works well in Nuendo and Cubase on a 7.1.2 bed. I'm also using it on all sorts of surround reverb channels (7.0, quad, and 7.1 with Phoenixverb). Can't help you with Studio One, sorry

**Reply by 21stSentry** [reply]
I am using Cubase. How do you configure an instrument or effect with AG for a 7.1.2 bed?
I could only get SkyDust 3D in stereo and want to use AG for the heavy CPU surround reverbs like Tai Chi etc.
